pod/recorder/admin.py

```diff
@@ -18,5 +19,17 @@ class RecordingFileAdmin(admin.ModelAdmin):
     list_filter = ('type',)
 
 
+class RecorderAdmin(admin.ModelAdmin):
+    def Description(self, obj):
+        return mark_safe('%s' % obj.description)
+
+    list_display = (
+        'name', 'Description', 'address_ip', 'user', 'type', 'directory')
+    list_display_links = ('name',)
+    list_filter = ('user',)
```

Comment:
We have too many user to filter with... Please remove this :(

Reply:
Fixed

Reply:
ok